pkg/controller/operators/catalog/operator.go

```diff
@@ -149,7 +151,9 @@ func NewOperator(ctx context.Context, kubeconfigPath string, clock utilclock.Clo
 	}
 
 	// create a config that validates we're creating objects with labels
-	validatingConfig := validatingroundtripper.Wrap(config)
+	_ = apiextensionsv1.AddToScheme(scheme)   // required by opClient
+	_ = apiregistrationv1.AddToScheme(scheme) // required by opClient
```

Comment:
Does this mutate the manager's scheme, or is this mutating a copy for use only by `NewOperator`? If the former (as I suspect), I think I'd put this in `main` because it would apply to anything else that uses the manager's scheme.

Reply:
Main just passes the standard scheme from client-go. Though, I think your comment still applies.